arm-network/2017-06-01/swagger/applicationGateway.json

```diff
@@ -390,7 +390,7 @@
         "tags": [
           "ApplicationGateways"
         ],
-        "operationId": "ApplicationGateways_ListAvailableSslOptions",
+        "operationId": "ApplicationGateways_AvailableSslOptions",
```

Comment:
@akshaysngupta can you link to it? If the SDK hasn't been generated/released (I assume it hasn't since this spec is new and under review), then you'd need to generate from the last version of the spec anyway (after merging this PR) and would need to re-run your tests to make sure things are working. Anyway, @balajikris can help evaluate, I'm just trying to provide information, so you can make a conscious decision.